Telegram/server.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * TODO(developer): 
+ * Add your service key to the current folder.
+ * Uncomment and fill in these variables.
+ */
+// const projectId = 'my-project';
+// const location = 'global';
+// const agentId = 'my-agent';
+// const languageCode = 'en'
+// const TELEGRAM_TOKEN='1234567898:ABCdfghTtaD8dfghdfgh45sdf65467M';
+// const SERVER_URL='https://example.com';
+const getUuid = require('uuid-by-string');
+const express = require('express');
+const axios = require('axios');
+const bodyParser = require('body-parser');
+
+const API_URL = `https://api.telegram.org/bot${TELEGRAM_TOKEN}`;
+const URI = `/webhook/${TELEGRAM_TOKEN}`;
+const WEBHOOK = SERVER_URL + URI;
+
+const app = express();
+app.use(bodyParser.json());
+
+
+// Imports the Google Cloud Some API library
+const {SessionsClient} = require('@google-cloud/dialogflow-cx');
+/**
+ * Example for regional endpoint:
+ *   const location = 'us-central1'
+ *   const client = new SessionsClient({apiEndpoint: 'us-central1-dialogflow.googleapis.com'})
+ */
+const client = new SessionsClient();
+
+/**
+ * Calls the Dialogflow CX API with the specific user query input and returns a string
+ * from the API with a response for the users query depending on the matched intent.
+ * The console.log() and console.info lines are included purely for debugging purposes
+ * and and don't impact the end users experience.
+ */
+async function detectIntentText(query,chatId) {
+  let agentResponse = '';
+
+  const sessionId = getUuid(chatId);
```

Comment:
https://www.npmjs.com/package/uuid-by-string
> You can specify the UUID version. Available versions is 3 and 5 according to RFC-4122. The version is responsible for the hashing algorithm: version 3 uses MD5, and version 5 uses SHA-1. SHA-1 used by default if version is not specified.

Looks like it uses v5 by default, and it may need to be explicitly specified for v3.